openfisca_france/parameters/prestations/minima_sociaux/ppa/montant_de_base.yaml

```
@@ -15,4 +15,6 @@ values:
     reference: https://www.legifrance.gouv.fr/eli/decret/2018/5/3/SSAA1805962D/jo/texte
   2018-08-01:
     value: 551.51
-    reference: https://www.legifrance.gouv.fr/eli/decret/2018/10/3/SSAA1822062D/jo/texte
\ No newline at end of file
+    reference: https://www.legifrance.gouv.fr/eli/decret/2018/10/3/SSAA1822062D/jo/texte
+  2020-04-01:
+    value: 553.17
```

```suggestion
    value: 553.17
    reference: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000041835204
```
Référence tirée des barèmes IPP. https://www.ipp.eu/baremes-ipp/prestations-sociales/3/minima_sociaux/pa_m/

---

Merci ! D'ailleurs, la référence indique un montant à `553.16`. Corrigé dans un nouveau commit.